build01/hydra.nix

```diff
@@ -135,14 +138,22 @@ in
       path = with pkgs; [ hydra-unstable netcat ];
       script = ''
         set -e
-        export HYDRA_ADMIN_PASSWORD=$(cat ${cfg.services.hydra.adminPasswordFile})
+        while IFS=; read -r user role password email; do
+          if [[ "$user" == "admin" ]]; then
+            export HYDRA_ADMIN_PASSWORD="$password"
+          fi
+          opts=("$user" "--role" "$role" "--password" "$password")
```

Comment:
Not a lot better but hydra-create-user has:

```
  [--password-hash SHA1-HASH]
```

Reply:
Could we export those hashes from the current database?

Reply:
done.